specs/error_state/error_return_data_out_of_bound.md

```diff
@@ -0,0 +1,27 @@
+# ErrorReturnDataOutOfBound state
+
+## Procedure
+this type of error only occurs when executing op code is `RETURNDATACOPY`.
+
+### EVM behavior
+when run `RETURNDATACOPY` op code, read stack for three fields: `memOffset`,
+`dataOffset`, `length`. then check if overflow as following order:  
+1. check `dataOffset`is u64 overflow, stop if yes.
+2. calculate `end` =  `dataOffset` + `length`.  
+3. check `end`is u64 overflow, stop if yes.
+4. check `end`> last callee's return data length, stop if yes.
```

Comment:
some small style problem..   

`end` =  `dataOffset` + `length` => 2 spaces.
check `end`is => no space
check `end`>  => no space